src/test/java/org/gridsuite/modification/server/utils/MatcherGeneratorCreationInfos.java

```diff
@@ -41,7 +41,31 @@ public boolean matchesSafely(GeneratorCreationInfos m) {
             || (m.getReactivePowerSetpoint() == null && reference.getReactivePowerSetpoint() == null))
             && m.isVoltageRegulationOn() == reference.isVoltageRegulationOn()
             && ((m.getVoltageSetpoint() != null && m.getVoltageSetpoint().equals(reference.getVoltageSetpoint()))
-            || (m.getVoltageSetpoint() == null && reference.getVoltageSetpoint() == null));
+            || (m.getVoltageSetpoint() == null && reference.getVoltageSetpoint() == null))
+            && ((m.getDroop() != null && m.getDroop().equals(reference.getDroop())))
```

Comment:
What about using `Objects.equals(a, b)` and leaving the `||` ?

Reply:
Done